src/pages/markets/basket-indices/index.tsx

```diff
@@ -12,16 +12,16 @@ const BasketPage = () => (
     <Layout type="noNav">
         <SEO
             description={localize(
-                'Trade Basket indices online with Deriv. Open a demo account and enjoy a wide range of currency pairs on our responsive and intuitive trading platforms.',
+                '_t_Trade Basket indices online with Deriv. Open a demo account and enjoy a wide range of currency pairs on our responsive and intuitive trading platforms._t_',
```

Comment:
After merging this https://github.com/niloofar-deriv/deriv-com/pull/86
we don't need to pass localize to SEO